docs/Microservices/RPC/buildRPC_Manual.md

```diff
@@ -0,0 +1,228 @@
+In addition to using GitHub Actions to obtain a binary, the RPC binary can also be manually built. The steps below walk through how to build RPC on Windows 10, Ubuntu (18.04 or 20.04), and CentOS7/8.
+
+### Required Software
+
+- [git](https://git-scm.com/downloads)
+
+Additionally, if using Windows 10:
+
+- [Microsoft Visual Studio*](https://visualstudio.microsoft.com/): 2019 or newer version of Visual Studio Community/Professional
+    - Make sure to install the **Desktop development with C++** package at time of installation or via the 'Get tools and extensions' menu within Microsoft Visual Studio*.
+
+<br>
+
+The steps below assume the following directory structure where rpc is the clone of the [rpc repository](https://github.com/open-amt-cloud-toolkit/rpc), vcpkg is a clone of the VCPKG tool source and build is the RPC build directory. Both vcpkg and build directories will be created in later steps.
+
+```
+\rpc
+  |__vcpkg
+  |__build
+```
+
+
+### Clone the Repository
+
+1. On your development system, navigate to a directory of your choice to clone and build RPC.
+
+2. Clone the RPC repository.
+
+    ``` bash
+    git clone https://github.com/open-amt-cloud-toolkit/rpc.git && cd rpc
+    ```
+
+### Install Prerequisites and Build RPC
+
+=== "Windows"
+    Open 'x64 Native Tools Command Prompt for VS 20XX' as Administrator on your development system.  **This is NOT a regular Windows Command Prompt.**  This specific tool is used for compiling the RPC executable.
+    
+    ![NTCP](../../assets/images/x64NativeToolsCP.png)
+
+    **Build VCPKG and C++ REST SDK**
+
+
+    1. In the `rpc` directory, clone the VCPKG repository. Vcpkg is a C/C++ Library Manager for Windows that was created by Microsoft.  Find out more about it [here](https://github.com/microsoft/vcpkg).
+    ``` bash
+    git clone --branch 2020.11-1 https://github.com/microsoft/vcpkg.git && cd vcpkg
+    ```
+
+    2. Build vcpkg.exe using the following command.
+    ``` bash
```

Comment:
Any reason for the "bash" directive? Is "```" enough?

Reply:
I think the ```bash directive is just so that mkdocs interprets the following command line stuff and displays it a certain way.

Reply:
Right, all it does is give syntax highlighting in code snippets. It's for consistency across command prompt cmds across the docs.